definitions/divorce/json/CaseEventToFields/CaseEventToFields-pet-sol-selects-own-org-nonprod.json

```diff
@@ -1,206 +0,0 @@
-[
-    {
-        "LiveFrom": "07/04/2020",
```

Comment:
moved to:
https://github.com/hmcts/div-ccd-definitions/pull/707/files#diff-80885e0fd6558cfe560b972365fef73a512693db28f4453010139a80958cf4e3R2359-R2562

It's part of RRJ release.